core/lib/instrument_api.h

```diff
@@ -4553,6 +4553,22 @@ DR_API
 bool
 dr_raw_tls_cfree(uint offset, uint num_slots);
 
+/**
+ * Provides raw access to offset \p offset from TLS base \p tls_register
```

Comment:
This should make it clear that this is talking about *raw* TLS slots ("raw access" does not suffice: it sounds like just direct access to non-raw slots).

Reply:
Done.